### PR TITLE
Map phar in stub file and set a versioned alias

### PR DESCRIPTION
**Pull request by jeskew**
Backporting the phar fix to the 2.8 branch.

/cc @mtdowling

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -23,13 +23,13 @@ view-docs:
...
-package: burgomaster
+package: clean burgomaster
 	time php build/packager.php
 
 # Downloads a copy of Burgomaster
-burgomaster:
+burgomaster: clean
 	mkdir -p build/artifacts
-	curl -s https://raw.githubusercontent.com/mtdowling/Burgomaster/0.0.2/src/Burgomaster.php > build/artifacts/Burgomaster.php
+	curl -s https://raw.githubusercontent.com/mtdowling/Burgomaster/master/src/Burgomaster.php > build/artifacts/Burgomaster.php
```
Comment (mtdowling) [inline Makefile]: I think you would at least want to lock to a commit sha here.

**Comment by mtdowling** [comment]
one minor piece of feedback, but LGTM

File: Makefile (hunk 1 above)
Comment (jeskew) [inline Makefile]: Any chance you could release a 0.0.3? I was going to leave it at master until the next release (but a sha would work just as well).

Comment (mtdowling) [inline Makefile]: Done https://github.com/mtdowling/Burgomaster/releases/tag/0.0.3

**Pull request merged**